source/board/mimxrt1170_evk.c

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file    mimxrt1170_evk.c
+ * @brief   board ID for the NXP MIMXRT1170-EVK board.
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2020, ARM Limited, All Rights Reserved
```

Comment:
Please change the copyright to your own (or NXP's, as appropriate) for 2020-2021.

Reply:
fixed as suggested.